tests/emukit/benchmarking/test_benchmark_result.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_array_equal
+
+from emukit.benchmarking.loop_benchmarking.benchmark_result import BenchmarkResult
+
+
+def test_benchmark_result_initialization():
```

Comment:
what about also testing for loop2 to make sure the init loops over the loop names

Reply:
done